src/main/kotlin/com/manimdsl/stylesheet/Stylesheet.kt

```diff
@@ -31,18 +32,20 @@ sealed class StylesheetProperty {
 data class AnimationProperties(
     override val borderColor: String? = null,
     override val textColor: String? = null,
+    override val hide: Boolean = false,
```

Comment:
Can a variable be hidden from the start and also during some animation?

Reply:
just for any variable

Reply:
No I mean does it need to be in `AnimationProperties` and `StyleProperties` or can it just be inside one of them?